lib/db.js

```diff
@@ -290,26 +290,20 @@ class LimitDBRedis extends EventEmitter {
   }
 
   takeElevated(params, callback) {
-    const valError = validateERLParams(params.elevated_limits);
-    if (valError) {
-      return callback(valError)
+    let erlParams;
+
+    if (params.elevated_limits) {
+      erlParams = utils.getERLParams(params.elevated_limits);
+      const valError = validateERLParams(erlParams);
```

Comment:
I think this was commented in another PR, but I think it's odd to validate _after_ parsing. 

It's fine for me in this PR, but we can take a look at this for future general refactoring.

Reply:
yeah, we could debate if parsing should come before or after validating. The thing is that parsing needs to be done for extracting the quota which comes in the form of 'quota_per_calendar_month', so it gets translated into `quota_amount` and `quota_interval`. That's why we're validating after parsing. But we could circle back into this